teamscale-client/src/main/java/com/teamscale/client/PrioritizableTestCluster.java

```diff
@@ -30,26 +30,20 @@ public class PrioritizableTestCluster {
 	 * compared to other scores of the same request. It makes no sense to compare the score against absolute values.
 	 * The value is 0 if no availableTests are given.
 	 */
-	@Json(name = "currentScore")
+	@JsonProperty("currentScore")
 	public double score;
 
 	/**
 	 * Field for storing the tests rank. The rank is the 1-based index of the test
 	 * in the prioritized list.
 	 */
-	@Json(name = "rank")
```

Comment:
You deleted this one, but kept the one on the `PrioritizableTest` - is there any difference? 🤔

Reply:
No, just overlooked it for `PrioritizableTest`. Removed it there as well